doc-content/enterprise-only/admin-and-config/business-central-editing-data-sources-proc.adoc

```diff
@@ -6,65 +6,59 @@
 From the *Data Source Authoring* page you can add data sources and database drivers to {CENTRAL}.
 
 [float]
-==== Adding a data source to {CENTRAL}
+== Adding a data source to {CENTRAL}
 
 .Procedure
-. Log in to {CENTRAL}.
-. Go to *Admin* -> *Data Sources*. The *Data Sources Authoring* page opens.
+. In {CENTRAL}, select the *Admin* icon in the top-right corner of the screen and select *Data Sources*. The *Data Sources Authoring* page opens.
```

Comment:
This change is based on Stetson's feedback copied below and the IBMSG 210p. If we use it throughout the docs then I do not think it would cause confusion. I will check and ensure "go to *Admin*" is not used. 

> What to call the BC settings icon and page is tricky because the actual cogwheel icon is called **Admin** when you hover over, but the page itself is called **Settings**. When I refer to the icon, I say "the **Admin** icon in the upper-right corner of the screen" etc.

Reply:
"go to *Admin*" is not used.